### ISSUE TITLE
RecyclerView scrollToPosition makes header go to wrong place

### ISSUE THREAD

**Issue by mazoni** [carlonzo/StikkyHeader#49]
Hi there, congrats for the library, it's really easy to use and works great.
I'm having a small issue: whenever I do a scrollToPosition on my recyclerview the sticky header don't refresh itself. In other words if I'm at the beginning of the recyclerview and execute a scrollToPosition to the last position of the recyclerview the whole header remains in screen, event the part that is not supposed to be sticky. I can attach some images if that is not clear enough. Can someone help me? Thanks in advance.

**Comment by mazoni** [comment]
It probably has something to do with this: http://stackoverflow.com/questions/27819507/recyclerview-scrolltoposition-not-trigger-scrolllistener ... since the stickyheader counts with OnScrollListener to update the header position...

**Comment by mazoni** [comment]
Guys, I've found a work around for this case. It's to use the smoothScrollToPosition within the layout manager. This seems to trigger the ScrollListener and the header works great then... 

I used to have this:
```java
recyclerView.scrollToPosition(recyclerAdapter.getItemCount() - 1);
```
and now I have this:
```java
recyclerView.getLayoutManager().smoothScrollToPosition(recyclerView, null, recyclerAdapter.getItemCount() - 1);
```

I'll close this issue since it seems to be a bug in Android and not on this library. And this workaround works fine.
